kvbc/src/kvbc_app_filter/kvbc_app_filter.cpp

```diff
@@ -422,7 +422,7 @@ std::optional<kvbc::categorization::ImmutableInput> KvbAppFilter::getBlockEvents
 kvbc::categorization::EventGroup KvbAppFilter::getEventGroup(kvbc::EventGroupId event_group_id, std::string &cid) {
   LOG_DEBUG(logger_, "getEventGroup " << event_group_id << " for " << client_id_);
   // get global_event_group_id corresponding to trid_event_group_id
-  const auto key = client_id_ + concordUtils::toBigEndianStringBuffer(event_group_id);
+  const auto key = client_id_ + "#" + concordUtils::toBigEndianStringBuffer(event_group_id);
```

Comment:
This needs a comment otherwise it is an arbitrary `#` :)

Reply:
Updated.